docs/tasks/debug-application-cluster/troubleshooting.md

```diff
@@ -33,7 +33,6 @@ and command-line interfaces (CLIs), such as [`kubectl`](/docs/user-guide/kubectl
 
 We also have a number of FAQ pages:
 
-   * [User FAQ](https://github.com/kubernetes/kubernetes/wiki/User-FAQ)
    * [Debugging FAQ](https://github.com/kubernetes/kubernetes/wiki/Debugging-FAQ)
```

Comment:
@bgrant0607 question for you - in the work around https://github.com/kubernetes/community/issues/116, are either of the Debugging FAQ or Services FAQ pages due to be removed as well? i.e. https://github.com/kubernetes/kubernetes/wiki/Debugging-FAQ or https://github.com/kubernetes/kubernetes/wiki/Services-FAQ?

Reply:
@heckj All of the wiki pages need to be removed eventually. Please extend your issue or file another to ensure that content that is still relevant is moved to actual docs.

Reply:
BTW, the community issue refers to community wiki pages. But we also need to get rid of the (even older) kubernetes/kubernetes wiki pages